opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/client/TracingAsyncRestTemplateInterceptor.java

```diff
@@ -64,52 +64,51 @@ public TracingAsyncRestTemplateInterceptor(Tracer tracer, List<RestTemplateSpanD
                                                           byte[] body,
                                                           AsyncClientHttpRequestExecution execution) throws IOException {
 
-        final Scope scope = tracer.buildSpan(httpRequest.getMethod().toString())
-                .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_CLIENT).startActive(false);
-        tracer.inject(scope.span().context(), Format.Builtin.HTTP_HEADERS, new HttpHeadersCarrier(httpRequest.getHeaders()));
+        final Span span = tracer.buildSpan(httpRequest.getMethod().toString())
```

Comment:
The span is no longer being activated before the request is executed - won't this cause a problem?

Reply:
Yes, it should be activated for the following interceptors